.gitignore

```diff
@@ -49,6 +49,9 @@ hs_err_pid*.log
 # Ignore MacOSX files.
 .DS_Store
 
+# Ignore the documentation build files in pydocs
+sdks/python/docs
+
 # NOTE: if you modify this file, you probably need to modify the file set that
```

Comment:
please take a peek at this note; can easily break source distribution / rat plugin.